src/Codeception/Module/Redis.php

```diff
@@ -42,7 +42,7 @@ class Redis extends CodeceptionModule
     protected $config = [
         'host'          => '127.0.0.1',
         'port'          => 6379,
-        'cleanupBefore' => 'suite'
+        'cleanupBefore' => 'test'
```

Comment:
I'd replace this with `cleanup` options:

* cleanup: false - disable cleanup
* cleanup: true - cleanup before test
* cleanup: test - same as above
* cleanup: suite - cleanup suite.